### PR TITLE
chore: Replace fonts related less vars with native css vars

### PR DESCRIPTION
**Pull request by przemvs**
## Description

Replacing fonts @ variables to var() less variables.

## Checklist

- [ ] mentions the JIRA issue in the PR name (Ex. [WPB-423])
- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

[WPB-423]: https://wearezeta.atlassian.net/browse/WPB-423?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/style/common/colors.less [hunk 1]
```diff
@@ -56,39 +56,39 @@
 }
 
 .font-size-xxs {
-  .font-size(@font-size-xxs);
+  .font-size(var(--font-size-xxs));
 }
 .font-size-xs {
-  .font-size(@font-size-xs);
+  .font-size(var(--font-size-xsmall));
```
Comment (aweiss-dev) [inline src/style/common/colors.less:62]: - Shouldn't after base be "small"?

File: src/style/common/variables.less [hunk 2]
```diff
@@ -298,25 +267,25 @@ body {
 }
 
 @avatar-diameter-xl: 200px;
-@avatar-initials-xl: @font-size-xl;
+@avatar-initials-xl: var(--font-size-xxl);
```
Comment (aweiss-dev) [inline src/style/common/variables.less:270]: Why is xl xxl now?

**Review by e-maad — APPROVED** [review]
Great initiative 🚀 🏅 
It would be nice if you update the PR title (commit message while merging) to something meaningful, for example: Replace fonts related less vars with native css vars.

File: src/style/common/variables.less [hunk 3]
```diff
@@ -298,25 +267,25 @@ body {
...
 
 @avatar-diameter-l: 72px;
-@avatar-initials-l: @font-size-lg;
+@avatar-initials-l: var(--font-size-xlarge);
```
Comment (aweiss-dev) [inline src/style/common/variables.less:273]: Same for large

File: src/style/components/asset/controls/media-button.less [hunk 4]
```diff
@@ -91,7 +91,7 @@
   .media-button-lg & {
     .circle(@media-button-size-lg);
 
-    font-size: @font-size-xl;
+    font-size: var(--font-size-xxl);
```
Comment (aweiss-dev) [inline src/style/components/asset/controls/media-button.less:94]: XXL?

**Review by aweiss-dev — APPROVED** [review]

**Pull request merged**